src/screens/InspectionCapture/index.js

```diff
@@ -16,6 +16,7 @@ import useSnackbar from '../../hooks/useSnackbar';
 import useFullscreen from './useFullscreen';
 
 const enableComplianceCheck = false;
+const currencyCharacter = '€';
```

Comment:
instead of doing this and passing the character in the route, you can hardcode the prop that is given to the inspectionreport component